app/uk/gov/hmrc/fileupload/ApplicationModule.scala

```diff
@@ -73,10 +72,7 @@ class ApplicationModule @Inject()(
 
   private val logger = Logger(getClass)
 
-  lazy val db = reactiveMongoComponent.mongoConnector.db
-
-  implicit val reader = new UnitOfWorkReader(EventSerializer.toEventData)
-  implicit val writer = new UnitOfWorkWriter(EventSerializer.fromEventData)
+  lazy val db: MongoComponent = mongoComponent
```

Comment:
small thing, maybe remove the lazy db in favour of referencing the mongoComponent in this file.  I see mixed use of db and mongoComponent. If you prefer the lazy db all good, then change the usage of mongoComponent to db.

Reply:
Done. Pushed a new commit.